cli/templatecreate.go

```diff
@@ -107,6 +107,14 @@ func (r *RootCmd) templateCreate() *clibase.Cmd {
 
 			message := uploadFlags.templateMessage(inv)
 
+			var varsFiles []string
+			if !uploadFlags.stdin() {
+				varsFiles, err = DiscoverVarsFiles(uploadFlags.directory)
+				if err != nil {
+					return err
+				}
```

Comment:
I could imagine a case where someone had tfvars files lying around unused before, worked around the issue, and left them there. Now we're going to auto-discover them. I'm not sure what this will break.
Should at the very least add an info message about the auto-discovered vars files?

Reply:
Ok, I changed the code to print a message alerting about the presence of tfvars 👍